From 268aa123fa7f67f689deb130ec8b4c51d80ffcc6 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 19 Nov 2014 16:12:57 -0500
Subject: [ethernet] be2net: fix sparse warnings in be_cmd_req_port_type{}

Message-id: <1416413587-12091-10-git-send-email-ivecera@redhat.com>
Patchwork-id: 100343
O-Subject: [RHEL7.1 PATCH 09/19] be2net: fix sparse warnings in be_cmd_req_port_type{}
Bugzilla: 1165755
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: #1165755

Upstream commit(s):
commit 72d7e2bf8f4c4f126b352bfbb67a64348623e31e
Author: Suresh Reddy <Suresh.Reddy@emulex.com>
Date:   Fri Sep 19 15:46:52 2014 +0530

    be2net: fix sparse warnings in be_cmd_req_port_type{}

    This patch fixes a sprase warnings regarding endian declarations introduced
    by the following commit:

    fixes: e36edd9 ("be2net: add ethtool "-m" option support")

    Signed-off-by: Suresh Reddy <Suresh.Reddy@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_cmds.h b/drivers/net/ethernet/emulex/benet/be_cmds.h
index e86a5ef..b064c50 100644
--- a/drivers/net/ethernet/emulex/benet/be_cmds.h
+++ b/drivers/net/ethernet/emulex/benet/be_cmds.h
@@ -1005,8 +1005,8 @@ struct be_cmd_resp_link_status {
 /*    Identifies the type of port attached to NIC     */
 struct be_cmd_req_port_type {
  struct be_cmd_req_hdr hdr;
- u32 page_num;
- u32 port;
+ __le32 page_num;
+ __le32 port;
 };
 
 enum {
-- 
1.7.1